documents/postman/README.md

```diff
@@ -0,0 +1,59 @@
+# Postman
+
+To download Postman, go [here](https://www.postman.com/downloads/). Postman allows you to send requests.
+
+## Postman collection
+
+The intention of this collection is to provide quick, easy functionality to send email, sms, and application notifications from non-admin users. As a non-admin user one can send a notification and get information regarding the notification. Creation, viewing and editing of templates should be done in the portal and not through the API.  
```

Comment:
- "application notifications" should be push notifications
- Can we change the mention of "non-admin users" to be more user friendly? This collection can be used by **business line users** integrating with VA Notify to get familiar with the API and send test notifications. 
- In the last sentence I would adjust to: "Creation, viewing and editing of templates can be done in the Self Service Portal."